pontoon/base/tests/test_models.py

```diff
@@ -0,0 +1,75 @@
+from django_nose.tools import assert_equal
+
+from pontoon.base.models import Translation
+from pontoon.base.tests import (
+    assert_redirects,
+    EntityFactory,
+    LocaleFactory,
+    ProjectFactory,
+    ResourceFactory,
+    TranslationFactory,
+    UserFactory,
+    TestCase
+)
+from pontoon.base.utils import aware_datetime
+
+
+class LocaleTests(TestCase):
```

Comment:
This class should be called `TranslationQuerySetTests` since that's the class being tested.